Cargo.toml

```diff
@@ -51,6 +51,7 @@ rustc-ap-rustc_target = "272.0.0"
 rustc-ap-syntax = "272.0.0"
 rustc-ap-syntax_pos = "272.0.0"
 failure = "0.1.1"
+bytecount = "0.3.2"
```

Comment:
Should we opt-in to the simd feature? I don't think any Rust host env is missing simd nowadays.